pkg/sfu/downtrack.go

```diff
@@ -641,10 +641,12 @@ func (d *DownTrack) postMaxLayerNotifierEvent() {
 		return
 	}
 
+	d.bindLock.Lock()
```

Comment:
It would send to closed channel too, we need to move check `IsClosed()` under the lock

Reply:
I had it that way and reverted it. My bad 🤦 . Thank you @cnderrauber

Reply:
Addressing it in this commit - https://github.com/livekit/livekit/pull/2179/commits/572daa1e1c6c2e7c94dd84625002c4732f97b87a